scripts/RetentionScript/README.md

```diff
@@ -0,0 +1,68 @@
+ ```
+ 4) Filtering for parition with partitionId "18cf9495-7233-42a0-929d-5ca9c110b861"
+```powershell
+ .\RetentionScript.ps1 -ContainerName "brstorage"-DateTimeBefore "2018-06-18 23.44.03Z" -StorageAccountName "storageaccountname" -StorageAccountName "storgeAccountName" -ClusterEndPoint "clustername.centralus.cloupapp.azure.com:19080" -SSLCertificateThumbPrint "Client#Certificate#Thumbpring" -PartitionId "18cf9495-7233-42a0-929d-5ca9c110b861"
+ ```
+ 
+  How to use with File Storage:
+  If the storage is secured with username(in the format Domain\user) and password, then, you need to provide password as secured string to the input variable "Pass"
+  Examples: 
+  The below example  will delete backups of  the paritition with partition id "18cf9495-7233-42a0-929d-5ca9c110b861" before "2018-06-18 23.44.03Z".
+```powershell
+$pass = "Passoword" | ConvertTo-SecureString -AsPlainText -Force
+ .\RetentionScript.ps1 -DateTimeBefore "2018-06-18 23.44.03Z" -UserName "Domain\brsuser"  -ClusterEndPoint "clustername.centralus.cloupapp.azure.com:19080" -SSLCertificateThumbPrint "Client#Certificate#Thumbpring" -Pass $pass -FileSharePath "\\fileshare\sharedfolder" -PartitionId "18cf9495-7233-42a0-929d-5ca9c110b861"  
+ ```
+ 
+  The below example  will delete backups of  the application with application id "WebReferenceApplication~RestockRequestManager" before "2018-06-18 23.44.03Z".
+```powershell
+ .\RetentionScript.ps1 -DateTimeBefore "2018-06-18 23.44.03Z" -UserName "Domain\brsuser"  -ClusterEndPoint "clustername.centralus.cloupapp.azure.com:19080" -SSLCertificateThumbPrint "Client#Certificate#Thumbpring" -Pass $pass -FileSharePath "\\fileshare\sharedfolder" -ApplicationId "WebReferenceApplication~RestockRequestManager"
+ ```
+ 
+  The below example  will delete backups of  the service with service id "WebReferenceApplication~RestockRequestManager" before "2018-06-18 23.44.03Z". 
+ ```powershell
+ .\RetentionScript.ps1 -DateTimeBefore "2018-06-18 23.44.03Z" -UserName "Domain\brsuser"  -ClusterEndPoint "clustername.centralus.cloupapp.azure.com:19080" -SSLCertificateThumbPrint "Client#Certificate#Thumbpring" -Pass $pass -FileSharePath "\\fileshare\sharedfolder" -ServiceId "WebReferenceApplication~RestockRequestManager"
+ ```
+
+  For public file shares, do not provide any username while running the script
+ 
+  
+ Notes:
+ 1) DateTimeBefore should be provided in the format("yyyy-MM-dd HH.mm.ssZ")
+ 2) ClientCertificateThumbprint is required if you are cleaning up storage of a secured cluster.
+ 3) If the management end point of the cluster is this "https://cluster.centralus.cloudapp.azure.com:19080/", then, the clusterendpoint will be "cluster.centralus.cloudapp.azure.com:19080"
```

Comment:
So whether to use http or https you figure it out yourself?

Reply:
if it is https, you would need to provide clientcertificate